ais/common/urls.py

```diff
@@ -13,7 +13,7 @@
     url(r'^api/', include('api.urls')),
     url(r'^banquet/', include('banquet.urls')),
     url(r'^$', login_redirect),
-    url(r'^fairs/(?P<year>\d+)/', include('urls.urls')),
+    url(r'^fairs/', include('urls.urls')),
```

Comment:
Changed made so we can caught the missing path parameter when not entering a year.